utah-design-system-website/src/react/components/websiteContent/library/components/containers/TabGroup/TabGroupPropsDocumentation.jsx

```diff
@@ -0,0 +1,277 @@
+                  Which tab should be selected by default.<br />
+                  Must match the corresponding tab <code>id</code>.
+                </TableCell>
+              </TableRow>
+              <TableRow>
+                <TableCell><code className="primary-color">isVertical</code></TableCell>
+                <TableCell><code>boolean</code></TableCell>
+                <TableCell><code>false</code></TableCell>
+                <TableCell>
+                  Change the orientation of the tab group.
+                </TableCell>
+              </TableRow>
+              <TableRow>
+                <TableCell><code className="primary-color">onChange</code></TableCell>
+                <TableCell>
+                  <div className="props-code-wrapper">
+                    <code>function</code>
+                  </div>
+                </TableCell>
+                <TableCell>null</TableCell>
+                <TableCell>The function to call when a tab is clicked.</TableCell>
+              </TableRow>
+            </TableBody>
+          </Table>
+        </TableWrapper>
+      </div>
+
+      <h3><code>TabGroupTitle</code></h3>
+      <div className="documentation-content--small-text static-example static-example--blank">
+        <TableWrapper className="static-example__component-wrapper">
+          <Table className="table--lines-x table--full-width">
+            <TableHead>
+              <TableHeadRow>
+                <TableHeadCell className="text-left">Property</TableHeadCell>
+                <TableHeadCell className="text-left">Type</TableHeadCell>
+                <TableHeadCell className="text-left">Default</TableHeadCell>
+                <TableHeadCell className="text-left">Description</TableHeadCell>
+              </TableHeadRow>
+            </TableHead>
+            <TableBody>
+              <TableRow>
+                <TableCell><code className="primary-color">children</code></TableCell>
+                <TableCell>
+                  <div className="props-code-wrapper">
+                    <code>node</code>
+                  </div>
+                </TableCell>
+                <TableCell><em>required</em></TableCell>
+                <TableCell>
+                  The content of the title.
+                </TableCell>
+              </TableRow>
+              <TableRow>
+                <TableCell><code className="primary-color">className</code></TableCell>
+                <TableCell><code>string</code></TableCell>
+                <TableCell>null</TableCell>
+                <TableCell>
+                  The CSS Class to use on the title.
+                </TableCell>
+              </TableRow>
+              <TableRow>
+                <TableCell><code className="primary-color">tagName</code></TableCell>
+                <TableCell><code>string</code></TableCell>
+                <TableCell><code>div</code></TableCell>
+                <TableCell>
+                  What HTML element should the title be.
```

Comment:
This is phrased as a question so should probably end with a question mark? Or maybe be changed to something like: 'The HTML element used to wrap around the title.`